wp-content/themes/workingnyc/views/employ-nyc-landing-page.twig

```diff
@@ -2,27 +2,26 @@
 
 {% block content %}
 <div class="bg-scale-2">
-  <main class="bg-scale-0" id="content">
-    <div class="site-max-width bg-scale-0">
-      {% include ['components/utility-nav.twig'] with {
-        return: {
-          'label': __('Home', 'WNYC'),
-          'link': site.url
-        },
-        web_share: {
-          'title': post.title,
-          'text': (post.web_share_text) ? post.web_share_text,
-          'url': fn('wp_get_shortlink')
-        }
-      } %}
-    </div>
-
-    {# Header Section #}
-    <div class="site-max-width">
-      <header class="o-header">
+  <main class="bg-[#E9F4FD]" id="content">
+   {# Header Section #}
+    <div>
+      <header class="site-max-width pt-7 pb-[72px]">
```

Comment:
- [ ] The previous padding was correct (py-5 tablet:py-6 desktop:py-7)

Reply:
OK I am changing this, as you said it will be same for all screens that's why I removed for tablet and desktop and made it default.